vitaes-webapp/src/Builder.jsx

```diff
@@ -186,8 +186,8 @@ class CvItemForm extends Component {
     }
     for (const item in toAdd) {
       if (item.endsWith('date') && toAdd[item]) {
-        if (!toAdd[item].match(/^\d{4}-\d{2}-\d{2}$/)) {
-          toast.error(`Wrong format:${item}`);
+        if (!validateDate(item).match(/^\d{4}-\d{2}-\d{2}$/)) {
```

Comment:
That makes entirely no sense here, I think you meant:
```suggestion
        if (!validateDate(toAdd[item])) {
```